src/views/forms/NewMarketForm/NewMarketSelectionStep.tsx

```diff
@@ -0,0 +1,457 @@
+        ]}
+        label={stringGetter({ key: STRING_KEYS.MARKETS })}
+      >
+        {assetToAdd ? (
+          <Styled.SelectedAsset>
+            {assetToAdd?.assetName ?? assetToAdd.baseAsset} <Tag>{assetToAdd?.baseAsset}-USD</Tag>
+          </Styled.SelectedAsset>
+        ) : (
+          'e.g. "BTC-USD"'
+        )}
+      </SearchSelectMenu>
+      {assetToAdd && (
+        <>
+          <div>{stringGetter({ key: STRING_KEYS.POPULATED_DETAILS })}</div>
+          <div>
+            <Styled.Root value={tempLiquidityTier} onValueChange={setTempLiquidityTier}>
+              <Styled.Header>
+                {stringGetter({ key: STRING_KEYS.LIQUIDITY_TIER })}
+                <Styled.ButtonRow>
+                  <Button
+                    shape={ButtonShape.Pill}
+                    onClick={() => {
+                      if (canModifyLiqTier) {
+                        setTempLiquidityTier(liquidityTier);
+                      }
+                      setCanModifyLiqTier(!canModifyLiqTier);
+                    }}
+                  >
+                    {canModifyLiqTier ? (
+                      stringGetter({ key: STRING_KEYS.CANCEL })
+                    ) : (
+                      <>
+                        {stringGetter({ key: STRING_KEYS.MODIFY })}{' '}
+                        <Icon iconName={IconName.Pencil} />
+                      </>
+                    )}
+                  </Button>
+                  {canModifyLiqTier && (
+                    <Button
+                      shape={ButtonShape.Pill}
+                      action={ButtonAction.Primary}
+                      onClick={() => {
+                        setLiquidityTier(tempLiquidityTier);
+                        setCanModifyLiqTier(false);
+                      }}
+                    >
+                      {stringGetter({ key: STRING_KEYS.SAVE })}
+                    </Button>
+                  )}
+                </Styled.ButtonRow>
+              </Styled.Header>
+
+              {Object.keys(LIQUIDITY_TIERS).map((tier) => {
+                const { maintenanceMarginFraction, impactNotional, label, initialMarginFraction } =
+                  LIQUIDITY_TIERS[tier as unknown as keyof typeof LIQUIDITY_TIERS];
+                return (
+                  <Styled.LiquidityTierRadioButton
+                    key={tier}
+                    value={Number(tier)}
+                    selected={Number(tier) === tempLiquidityTier}
+                    disabled={!canModifyLiqTier}
+                  >
+                    <Styled.Header style={{ marginLeft: '1rem' }}>
+                      {label}
+                      {Number(tier) === assetToAdd?.liquidityTier && (
+                        <Tag style={{ marginLeft: '0.5ch' }}>
+                          ✨ {stringGetter({ key: STRING_KEYS.RECOMMENDED })}
+                        </Tag>
+                      )}
+                    </Styled.Header>
+                    <Styled.Details
```

Comment:
umm on mobile can they be a row / grid style? the vertical margin is pretty tall
<img width="544" alt="image" src="https://github.com/dydxprotocol/v4-web/assets/9400120/a594a772-1f4f-4d0d-8968-bf1be7613ee2">

Reply:
`grid` works!

<img width="362" alt="Screen Shot 2024-01-30 at 11 28 30 AM" src="https://github.com/dydxprotocol/v4-web/assets/13111994/f16faaa0-0b46-4463-a68f-2917cc7e29ee">

Reply:
`grid` works!

<img width="362" alt="Screen Shot 2024-01-30 at 11 28 30 AM" src="https://github.com/dydxprotocol/v4-web/assets/13111994/f16faaa0-0b46-4463-a68f-2917cc7e29ee">